helm/metrics-server-app/templates/priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.e2e }}
```

Comment:
Previously EnsureTillerInstalled created the priority class now we need this template.